hail/python/hailtop/aiotools/fs.py

```diff
@@ -608,6 +609,10 @@ async def copy_as_file(self,
             self.src_is_file = False
             await self.release_barrier()
             return
+        except:
```

Comment:
Can I ask, what unhandled error did we see here?  Just so I can keep this case in mind in the future.

Reply:
Lindo's service account didn't have permissions to access the files. So he got a google 403 exception. But the error message was `assert pending == 0`